### ISSUE TITLE
StackBlitz demo is dead

### ISSUE THREAD

**Issue by ummahusla** [ngrx/platform#956]
I've tried to access [`ExampleApp` demo via StackBlitz](https://github.com/ngrx/platform/tree/master/example-app#try-it-on-stackblitz) but the URL is broken

```
Something went wrong with this repository.

If you think this is a bug, please file an issue in our repo!
 Go back
```

**Comment by tdeschryver** [comment]
Seems like we're not the only ones with this problem.
I ,entioned it at stackblitz/core#354

**Comment by EricSimons** [comment]
We're investigating this first thing when we get to our office in a hr or so- will report back once we nail down the culprit!

**Comment by EricSimons** [comment]
Found it! It looks like the package.json has a dep that points to a bare git repo, specifically:

```
"@ngrx/db": "github:brandonroberts/ngrx-db-builds"
```

Due to github api limitations, we currently only support github repos that specify a specific release tag- for ex:

```
"@angular/flex-layout": "https://github.com/angular/flex-layout#2.0.0-beta.11-b8a652d"
```

The good news is that we've sorted how we're going to enable full support for bare github repos (+ specific commits/etc) and it'll be coming within the next month or so. Until then, would either of the following options work:

1. Point that @ngrx/db at a npm repo/specific release on gh
2. Temporarily have us override @ngrx/db in our import resolver to point at a specific release/etc, until we fully support bare github repos as deps

Most ideal on our end would be # 1, but happy to do # 2 if necessary!
